src/object/cli_obj.c

```diff
@@ -2256,7 +2288,7 @@ obj_req_fanout(struct dc_object *obj, struct obj_auxi_args *obj_auxi,
 					shard_task_reset_param, &reset_arg);
 			}
 			goto task_sched;
-		} else {
+		} else if (!require_shard_task) {
```

Comment:
How to handle a retry RPC with "require_shard_task" as true?

Reply:
It's not clear indeed. I'm not sure about what the best approach is. Shall we discuss this?

Reply:
We had a discussion on this. I'll think further, especially considering the new development related to HLCT.